doc/kubernetes/modules/ROOT/pages/storage-configurations.adoc

```diff
@@ -3,10 +3,7 @@
 
 {description}
 
-Keycloak provides two storage implementations:
-
-* Legacy Storage
-* New Storage (also called Map Storage)
+This setup supports different databases.
```

Comment:
currently it doesn't, as it only supports `postgres` right @ahus1 ?
```suggestion
This setup supports only PostgreSQL database.
```

Reply:
I counted Aurora as a second database :-)